tests/orchestrator/test_sequential_orchestrator.py

```diff
@@ -187,6 +187,42 @@ def generator_function_new_guid(context):
     outputs.append(str(output3))
     return outputs
 
+def generator_function_call_avtivity_with_name(context):
```

Comment:
nit: there's a typo here.
Mind changing this from "generator_function_call_avtivity_with_name" to "generator_function_call_activity_with_name"